pkgs/by-name/xo/xonsh/xontribs/xontrib-broot/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  pkgs,
+  python3,
+}: let
+  pname = "xontrib-broot";
+  version = "0.2.1";
+in
+  python3.pkgs.buildPythonPackage {
+    inherit pname version;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "jnoortheen";
+      repo = "xontrib-broot";
+      rev = "6f658ff88aba27b921017297d8c2c3dfb2ffa332";
+      sha256 = "sha256-9GqsTVCMvrWpTopHtEdicTyYRQzP1NVtQHZsfBT+fUg=";
+    };
+
+    doCheck = false;
+
+    format = "pyproject";
+
+    build-system = with pkgs.python3Packages; [
+      setuptools
+      pdm-pep517
+      poetry-core
+      pkgs.xonsh
```

Comment:
I've found that most of the xontribs can't actually depend on Xonsh without causing a circular dependency. And, in Nix, we don't actually need them installed.